src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -226,16 +227,62 @@ public override async Task<dynamic> GetHostingEnvironmentPostBody(string hosting
             return hostingEnvironmentPostBody;
         }
 
+        public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceName, string query)
+        {
+            if (!query.StartsWith("\""))
+            {
+                // BUG: Passing as JSON requires query to be wrapped in quotes, fix API expected content type
+                query = $"\"{query}\"";
+            }
+
+            var request = new HttpRequestMessage(HttpMethod.Post, $"/api/service/{cloudServiceName}/invokesql?api-version=2.0")
+            {
+                Content = new StringContent(query, Encoding.Default, "application/json")
+            };
+
+            var response = await SendObserverRequestAsync(request);
+            var result = await response.Content.ReadAsStringAsync();
+
+            try
+            {
+                response.EnsureSuccessStatusCode();
+            }
+            catch (HttpRequestException)
+            {
+                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider",
```

Comment:
Shouldn't this be logged as an error instead of info level?

Reply:
The exception will be logged by the dataProviderDecorator up the stack; this is just supplemental logging for reproducing the error.